src/web/components/ApiKeyManagement/ApiRolesCell.tsx

```diff
@@ -0,0 +1,26 @@
+import { ApiRoleDTO } from '../../../api/entities/ApiRole';
+import { Tooltip } from '../Core/Tooltip';
+
+import './ApiRolesCell.scss';
+
+type ApiRolesProps = {
+  apiRoles: ApiRoleDTO[];
+  showRoleTooltip?: boolean;
+};
+function ApiRolesCell({ apiRoles, showRoleTooltip: showRole = false }: ApiRolesProps) {
```

Comment:
can probably get rid of this renaming (guessing it was done by VS code refactoring?).
So just `showRoleTooltip = false`

Reply:
Yeah it was. Sorry I missed that.